ketchup/main.py

```diff
@@ -1,43 +1,62 @@
+from dateutil import parser
 import datetime as dt
 from ketchup.luffy import LUFFY
-import ketchup.day_counter as dc
+import ketchup.watch_schedule_builder as dc
 import ketchup.fun_fact as fun_fact
 
 
 def run():
-    date_format = "%Y,%m,%d"
-
-    episodes = input(
-        "Hello! How many episodes is the show you are interested in watching? "
-    )
+    episodes = int(input(
+        "Hello! How many episodes are in the show you are interested in watching? "
+    ))
 
     end_date = input(
         "What date would you like to have completed this series?(yyyy,mm,dd) "
```

Comment:
Users are able to input a date that has expired.

<img width="1207" alt="Screenshot 2023-08-13 at 17 19 02" src="https://github.com/chellygel/AnimeKetchup/assets/121133431/9c2603f0-7c4f-45a4-afc8-4e00dcdc7a35">

Image provided as an example. 

I feel as if it would be cleaner to only allow the "I'm sorry, but that limit and end date make it impossible!" to be output to the user in this circumstance, however, providing the user with option two, which gives them the ability to see when they could finish watching their show by, is a worthwhile addition. So that could remain also.

Reply:
could you elaborate on your solution with an example perhaps?

Reply:
addressed date math

Reply:
If user enters a past date and wants to try and watch the show, it will try and provide both options to the user. 

Option 1 and Option 2.

Option 1 says that they could watch -34 episodes a day to catch up. I feel as if this is redundant to specify to the user. Where instead, they can go ahead and be just told the second option by default.

I expect to see: 

```
What date would you like to have completed this series? (yyyy, mm, dd): {enter past date}
Will you start watching today? [Y/N]
What is the limit of episodes you will watch per day? [Leave empty or a number]
Wow, that's only -{x amount of days} until then!
I'm sorry, but that limit makes it impossible.
Don't worry, consider these options instead:
Option: You could choose to watch {x} episodes per day and finish by {y} 
Alternatively for you to watch Utena, you would have to watch {x} episode per day and finish by {y} instead.
```